source/TuviRestClientLib/Client.cs

```diff
@@ -54,10 +55,21 @@ protected async Task<HttpStatusCode> SendAsync(Message message, CancellationToke
 
             using (var request = await message.CreateRequestAsync(_baseUri, cancellationToken).ConfigureAwait(false))
             {
-                using (var response = await _httpClient.SendAsync(request, cancellationToken).ConfigureAwait(false))
-                {
-                    await message.CreateResponseAsync(response, cancellationToken).ConfigureAwait(false);
-                    return response.StatusCode;
+                try
+                {
+                    using (var response = await _httpClient.SendAsync(request, cancellationToken).ConfigureAwait(false))
+                    {
+                        await message.CreateResponseAsync(response, cancellationToken).ConfigureAwait(false);
+                        return response.StatusCode;
+                    }
+                }
+                catch (HttpRequestException ex)
```

Comment:
You can use [when](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/when#when-in-a-catch-clause) keyword

Reply:
done